Fabric.Identity.API/Startup.cs

```diff
@@ -196,8 +192,10 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             var options = app.ApplicationServices.GetService<IdentityServerAuthenticationOptions>();
             app.UseIdentityServerAuthentication(options);
             app.UseMvcWithDefaultRoute();
+
+            var healthCheckService = app.ApplicationServices.GetService<IHealthCheckerService>();
```

Comment:
Maybe change this to GetRequiredService so it throws an exception if IHealthCheckerService is not registered? That way we fail faster - on startup rather than when /_monitor/deep is called.